### ISSUE TITLE
kube-bench/cfg/cis-1.6/master.yaml 1.2.26 cannot pass

### ISSUE THREAD

**Issue by jonipatr** [aquasecurity/kube-bench#1245]
**Overview**

1.2.26 Ensure that the --request-timeout argument is set
The test is scored but there is no test added after initial grepping.

**What happened?**

Test always returns:
`[WARN] 1.2.26 Ensure that the --request-timeout argument is set as appropriate (Automated)`

**What did you expect to happen:**

Test should return:
`[PASS] 1.2.26 Ensure that the --request-timeout argument is set as appropriate (Automated)`
when --request-timeout is set to kube-apiserver.yaml

**Environment** 

```
- Kubernetes version: Kubernetes v1.24.2
- kube-bench config: cis-1.6
- kube-bench version: v0.6.8
```

**Running processes**

I'm not adding whole apiconfig but I can see that request-timeout is set.

```
ps -ef | grep api
...
--request-timeout=300s
...
```

**Reason**

Test 1.2.26 could be done like 1.2.28
```
      - id: 1.2.26
        text: "Ensure that the --request-timeout argument is set as appropriate (Automated)"
        audit: "/bin/ps -ef | grep $apiserverbin | grep -v grep"
        type: manual
        remediation: |
          Edit the API server pod specification file $apiserverconf
          and set the below parameter as appropriate and if needed.
          For example,
          --request-timeout=300s
        scored: true
```

```
      - id: 1.2.28
        text: "Ensure that the --service-account-key-file argument is set as appropriate (Automated)"
        audit: "/bin/ps -ef | grep $apiserverbin | grep -v grep"
        tests:
          test_items:
            - flag: "--service-account-key-file"
        remediation: |
          Edit the API server pod specification file $apiserverconf
          on the master node and set the --service-account-key-file parameter
          to the public key file for service accounts:
          --service-account-key-file=<filename>
        scored: true
```

in a nutshell it just checks that request-timeout is being set even to default like in the remediation text:

```
      - id: 1.2.26
        text: "Ensure that the --request-timeout argument is set as appropriate (Automated)"
        audit: "/bin/ps -ef | grep $apiserverbin | grep -v grep"
        tests:
          test_items:
            - flag: "--request-timeout"
        remediation: |
          Edit the API server pod specification file $apiserverconf
          and set the below parameter as appropriate and if needed.
          For example,
          --request-timeout=300s
        scored: true
```

**Comment by chen-keinan** [comment]
@jonipatr thanks for the input. you can pick up this issue if you wish

**Comment by jonipatr** [comment]
@chen-keinan 
request created: https://github.com/aquasecurity/kube-bench/pull/1246
